plugins/outputs/seq/seq.go

```diff
@@ -0,0 +1,136 @@
+package seq
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"github.com/influxdata/telegraf"
```

Comment:
nitpick: separate stdlib and other imports

Reply:
Fixed.